app/feed/page.tsx

```diff
@@ -1,94 +1,54 @@
-"use client";
-
-import GitHubEvent from "@/components/gh_events/GitHubEvent";
-import { IGitHubEvent, combineSimilarPushEvents } from "@/lib/gh_events";
-import { useEffect, useState } from "react";
-import { scrollTo } from "@/lib/utils";
 import LoadingText from "@/components/LoadingText";
-import { env } from "@/env.mjs";
+import { IGitHubEvent } from "@/lib/gh_events";
+import GitHubEvent from "@/components/gh_events/GitHubEvent";
+import { Octokit } from "octokit";
+import getGitHubAccessToken from "@/lib/getGitHubAccessToken";
+
+const GITHUB_PAT: string = getGitHubAccessToken() || "";
```

Comment:
use pre-authenticated instance of octokit that is exported in recent commit: https://github.com/coronasafe/leaderboard/commit/9dd8348a0d29eadb2abee109af59d41699ae75d9#diff-d6bae8eb0c0c7b194e2579597b05c4f079d6e7dcdf48141956793f18b3361f21